src/components/Composer/index.js

```diff
@@ -279,7 +279,12 @@ function Composer({
             }
 
             if (textInput.current !== event.target) {
-                return;
+                const isTargetInput = event.target.nodeName === 'INPUT' || event.target.nodeName === 'TEXTAREA' || event.target.contentEditable === 'true';
```

Comment:
Can you please add a comment to the code that explains _why_ this logic is here and why it is what it is?

Reply:
@tgolen something like this?

To make sure the composer does not capture paste events from other inputs, we check where the event originated. If it did originate in another input, we return early to prevent the composer from handling the paste

Reply:
That's wonderful!

Reply:
👍 added @tgolen